src/core/numerics/moments/moments.hpp

```diff
@@ -46,17 +46,17 @@ namespace core
             if constexpr (std::is_same_v<DepositTag, DomainDeposit>)
             {
                 auto& partArray = pop.domainParticles();
-                interpolate(std::begin(partArray), std::end(partArray), density, flux, layout);
+                interpolate(makeRange(partArray), density, flux, layout);
```

Comment:
given the changes to interpolator.hpp

this is much the same as just `interpolate(partArray, density, flux, layout);`

containers and ranges should be interchangeable I think

Reply:
seems ok